roles/sudoers/templates/92-ansible-managed

```diff
@@ -6,4 +6,4 @@
 #
 # Allow specific users or group to become the {{ item.user }} user.
 #
-{{ item.sudoers }}    ALL=({{ item.user }})    NOPASSWD:ALL
+{% if functional_admin_group is defined and functional_admin_group | length %}%{{ functional_admin_group }},{% endif %}{{ item.sudoers }}    ALL=({{ item.user }})    NOPASSWD:ALL
```

Comment:
I think it would be better to keep as much of the logic as possible in the plays, too keep the templates as simple as possible. What do you think?

Reply:
The alternative would be to add this ```functional_admin_group``` to each group for each cluster in the corresponding ```group_vars```. It keeps the template here more universal, but it is a lot of redundancy in the groupvars...

Reply:
In fact this is an attempt to take all logic with regard to sudoers out of the play, because the play requires root privileges, which we don't have. So we now create groups in the IDVault to manage who can sudo to what accounts. Not all groups on all clusters have been migrated to this logic yet, but the idea is that we can manage sudoers exclusively with the IDVault without root.